Lighthouse Camera/UI/Main/PreviewView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  PreviewView.swift
+//  Lighthouse Camera
+//
+//  Created by Punam Dahiya on 1/27/17.
+//  Copyright © 2017 Lighthouse. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+protocol PreviewViewDelegate: class {
+  func displayPreviewSession()
+}
+
+
+class PreviewView: UIView {
```

Comment:
Displays what the camera lens sees  in our UIView, previewView so that we should see a live "video" stream of phone camera's input within previewView